src/Core/Commander/Providers/TileProvider.js

```diff
@@ -60,6 +63,8 @@ TileProvider.prototype.executeCommand = function executeCommand(command) {
     parent.add(tile);
     tile.updateMatrix();
     tile.updateMatrixWorld();
+    tile.OBB().parent = tile;
```

Comment:
Unfortunately, we can't do that without breaking iTowns (browsetree goes through EVERY child of the node, so it crashes when it tries to process the OBB). Hopefully, #239 will allow us to do it. I left a comment in the meantime.